app/templates/main/index.html

```diff
@@ -1,8 +1,11 @@
 {% extends 'layouts/base.html' %}
-
+{# Some special functionality here: current_user represents the current user. is_authticated is false for an AnonymousUser
```

Comment:
thats fine, but you should remove the extra comments so we can merge this PR into master